pycytominer/annotate.py

```diff
@@ -78,7 +78,11 @@ def annotate(
     platemap = load_platemap(platemap, add_metadata_id_to_platemap)
 
     annotated = platemap.merge(
-        profiles, left_on=join_on[0], right_on=join_on[1], how="inner"
+        profiles,
+        left_on=join_on[0],
+        right_on=join_on[1],
+        how="inner",
+        suffixes=["_platemap", None],
```

Comment:
When would this be necessary? In our standard pipelines, the profiles we feed into `annotate()` do not contain any metadata info.

Are you using `annotate()` to add metadata info to profiles that already contain metadata?

Would it be better then to include a function argument (that defaults to False) to perform `external_merge_only`? Then you could use annotate without breaking existing functionality, and keep the changes you introduce in line 120

Reply:
This part of the PR addresses a minority use-case scenario and will have no impact/relevancy to most pycytominer users. I encountered it when began using pycytominer to analyze features profiles generated by PerkinElmer's Harmony software. The default profile files from Harmony contain a selection of Metadata columns including treatment_id, treatment_time_h and a few others. After standardizing the naming of the columns and using `annotate()` with the platemap I got an output with the hard-to-interpret list of columns: 
* `Metadata_treatment_id_x`
* `Metadata_treatment_id_y`
* `Metadata_treatment_time_h_x`
* `Metadata_treatment_time_h_y`
These column names result from the default `("_x", "_y")` suffixes option in [pandas.DataFrame.merge](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.merge.html). 

I wanted to keep both sets of columns in my data as a check to make sure that the data entered in Benchling (our Electronic Lab Notebook software) matched what was reflected in the microscope's data package. 

Replacing these default options with this PR will instead give the columns: 
* `Metadata_treatment_id`
* `Metadata_treatment_id_platemap`
* `Metadata_treatment_time_h`
* `Metadata_treatment_time_h_platemap`

This latter set of columns is much easier to make sense of and easier to code-for.